fronts-client/src/reducers/modalsReducer.ts

```diff
@@ -1,15 +1,16 @@
 import { Action } from 'types/Action';
 import { OptionsModalChoices } from 'types/Modals';
+import { ReactNode } from 'react';
 
 type OptionsModalState = null | {
   title: string;
-  description: string;
+  description: string | ReactNode;
```

Comment:
What prompted adding `ReactNode` here?

Reply:
ah yes interesting one.. 
It is defined with ReactNode in here: https://github.com/guardian/facia-tool/blob/master/fronts-client/src/types/Action.ts#L301

So it was type failing because it had two different definitions. A better fix would be to merge them in some way

Reply:
I'll try fix this

Reply:
ahhh yes, there should probably be a type for the modal, then the reducer and action could both use it?